aten/src/ATen/native/cuda/AveragePool3d.cu

```diff
@@ -321,6 +342,13 @@ void avg_pool3d_out_cuda_template(
   TORCH_CHECK((input.ndimension() == 4 || input.ndimension() == 5),
     "non-empty 4D or 5D (batch mode) tensor expected for input");
 
+  //if divisor=0 then we will ignore it
```

Comment:
`==` to indicate equality and a space after `//`.

Reply:
changed